plugin/build.gradle

```diff
@@ -189,7 +189,7 @@ integTest {
 }
 
 testClusters.integTest {
-    testDistribution = "ARCHIVE"
+    testDistribution = "INTEG_TEST"
```

Comment:
Why change to `INTEG_TEST`?

Reply:
If not we will get this error in integ_test:
```
| Caused by: java.lang.IllegalStateException: jar hell!
| class: com.amazonaws.encryptionsdk.CryptoMaterialsManager
| jar1: /local/home/seasonsg/dev/ml-commons/plugin/build/testclusters/integTest-0/distro/3.0.0-ARCHIVE/lib/aws-encryption-sdk-java-2.4.0.jar
| jar2: /local/home/seasonsg/dev/ml-commons/plugin/build/testclusters/integTest-0/distro/3.0.0-ARCHIVE/plugins/.installing-2105471977921602206/aws-encryption-sdk-java-2.4.0.jar
```

Reply:
hm.. I suspect we are going to see problems. 
`INTEG_TEST` does not pull in modules which might work for us for tests but when we have to run it with the distribution(release) it will have modules installed and we will end up with JarHell.
Could you manually try installing the plugin with OpenSearch 2.x locally assembled with this change? 
Can you verify we dont hit the problem ?

Reply:
I see. This fix is introduced by @HenryL27 so I think we need to involve him in the loop. If so do you have any idea on fixing this jarhell issue?

Reply:
Tried in my local, with all the changes in this PR, using distribution won't have jar hell issue.